src/api/controllers/user/get-shopping-list.js

```diff
@@ -0,0 +1,20 @@
+import { getUserById } from './helpers.js';
+
+async function getShoppingList(req, res, next) {
+  try {
+    const id = req.user.id;
+    const { shoppingList } = await getUserById(id);
+    if (!shoppingList) {
+      return res.status(401).json({ message: 'No recipes' });
+    }
+    return res.status(200).json({
+      resultMassage: '',
+      resultCode: '',
```

Comment:
We don't want empty messages and codes ❌

Reply:
I know. It's just a base thing!